### DISCUSSION TITLE
Сохранение состояния фильтра в виде нового queryTag-а

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1141] [Ideas]
Часто пользователи делают отборки данных под свои нужды, например: мои незавершённые заказы, или заказы на понедельник, или невыполненные задания со статусом срочно по подразделению х.  Каждому пользователю нужна возможность сохранять свои фильтры в качестве пресетов(быстрых фильтров), а не пользоваться только теми фильтрами что сделал админ.

**Comment by lee-to** [comment]
Это и есть суть квери тегов, вам нужно начать с основ разработки и научиться быть разработчиком, прокачать мышление и тогда подобные задачи вы будете решать без готового для вас сахара (который к слову не может быть всегда универсальным)
